### PR TITLE
feat(emqx_auth_jwt): use JWT for ACL checks

### PR DESCRIPTION
**Pull request by savonarola**

### CODE REVIEW

File: apps/emqx_authz/i18n/emqx_authz_schema_i18n.conf [hunk 1]
```diff
@@ -340,6 +340,28 @@ Commands can support following wildcards:\n
     }
   }
 
+  jwt {
```
Comment (savonarola) [inline apps/emqx_authz/i18n/emqx_authz_schema_i18n.conf:343]: We should also add Zh translation.

File: apps/emqx_authz/test/emqx_authz_jwt_SUITE.erl [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+    {ok, _} = emqtt:connect(C),
+
+    ?assertMatch(
+        {ok, #{}, [?RC_NOT_AUTHORIZED]},
+        emqtt:subscribe(C, <<"a/b">>, 0)
+    ),
+
+    ok = emqtt:disconnect(C).
+
+t_check_expire(_Config) ->
```
Comment (thalesmg) [inline apps/emqx_authz/test/emqx_authz_jwt_SUITE.erl:244]: Is it worth to also test `nbf` and `iat`?

Comment (savonarola) [reply apps/emqx_authz/test/emqx_authz_jwt_SUITE.erl:244]: with invalid `nbf` and `iat` a user can't authenticate... so seems that we should just stop authorizing after token expire.

**Comment by savonarola** [comment]
Failing CI tasks seem not be relevant to the PR

**Review by thalesmg — APPROVED** [review]

File: apps/emqx_authz/i18n/emqx_authz_schema_i18n.conf (hunk 1 above)
Comment (HJianBo) [reply apps/emqx_authz/i18n/emqx_authz_schema_i18n.conf:343]: Let me add it in another PR?

**Review by HJianBo — APPROVED** [review]

File: apps/emqx_authz/i18n/emqx_authz_schema_i18n.conf (hunk 1 above)
Comment (HJianBo) [reply apps/emqx_authz/i18n/emqx_authz_schema_i18n.conf:343]: see: https://github.com/emqx/emqx/pull/7769

**Pull request merged**